### PR TITLE
Features js dealing with firebase

### PR DESCRIPTION
**Pull request by chrisostome01**
### What this PR does

- Validation on login form and contact
- Getting user location
- the creation of a new article for admin
- View article
- delete blog
- read more about the blog

### How you may test it

1. clone this repo
2. use this branch 
3. Open the project
4. create an account and log in 

> But you will not be able to delete or upload a new post because you are not an admin

### Related Trello boards
[Validation for login and contact form](https://trello.com/c/SYmNkxED/15-validate-login-and-contact-forms)
[Getting user location](https://trello.com/c/Xvrnmgrw/16-get-user-location-using-javascript-geolocation-api)
[Creation of a new article for admin](https://trello.com/c/RrohSoe9/21-create-a-new-article)
[View list of articles](https://trello.com/c/fMlhptZj/17-view-a-list-of-their-articles)
[Delete a blog](https://trello.com/c/u1TRm1it/23-delete-an-existing-article)
[Adding a tool for editing HTML](https://trello.com/c/OyKGpflF/28-read-more-btn)
[Read more about the blog](https://trello.com/c/OyKGpflF/28-read-more-btn)

> validation
127 0 0 1_5500_ui_login html(iPhone 5_SE) (1)

127 0 0 1_5500_ui_blog html (4)

> view all article and deletion and post

127 0 0 1_5500_ui_browse html (2)

> after clicking read more 

127 0 0 1_5500_ui_blog html (3)

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -174,6 +178,33 @@ html{
...
+.user-profile {
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  align-items: center;  
+  border : 1px solid var(--underline);
+  border-radius: 100%;
+  box-shadow: 0 0 12px #181717a4;
```
Comment (renepromesse) [inline css/style.css]: define the variable of this box-shadow property and apply it everywhere you're using that color.

File: css/style.css [hunk 2]
```diff
@@ -174,6 +178,33 @@ html{
...
+  height: 100%;
+  transition: all 0.2s ease-in-out;
+  border-radius: 100%;
+}
+.user-profile img:hover{
+  width: 100%;
+  height: 100%;
+  border : none;
+  border-radius: 100%;
+  box-shadow: 0 0 12px #181717c7;
```
Comment (renepromesse) [inline css/style.css]: define a variable for this color too

File: js/provider.js [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+            notInfo != null ?  notInfo.innerHTML = '': '' ;
+            place.classList.remove(`${errorType}-msg`);
+        } ,4000);
+    }
+    else{
+        // setTimeout(() => {
+        //     msgTitleField != null ?  msgTitleField.innerHTML = '': '' ;
+        //     notInfo != null ?  notInfo.innerHTML = '': '' ;
+        //     place.classList.remove(`${errorType}-msg`);
+        // } ,lastIn);
```
Comment (renepromesse) [inline js/provider.js]: remove this comments if you're not using it

**Review by renepromesse — COMMENTED** [review]
@sezeranoJchrisostome this is good work. look for all feedback requested

File: css/style.css (hunk 1 above)
Comment (sezeranoJchrisostome) [reply css/style.css:190]: I worked on this in the second pr

File: js/provider.js (hunk 3 above)
Comment (sezeranoJchrisostome) [reply js/provider.js:31]: also this are in the second pr

**Pull request merged**